ext/standard/tests/strings/str_ends_with.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+str_ends_with() function - unit tests for str_ends_with()
+--FILE--
+<?php
+/* Prototype: bool str_ends_with (string $haystack string $needle );
```

Comment:
sorry but still missing a `,` just after `$haystack` (here and in `str_starts_with.phpt` too)
and also there's an extra space between `$needle` and `);` (here only)